sdk/src/main/java/com/uber/sdk/rides/client/internal/RetrofitUberRidesService.java

```diff
@@ -227,6 +228,15 @@ void setPlace(@Nonnull @Path("place_id") String placeId,
     void getRideDetails(@Nonnull @Path("request_id") String rideId, Callback<Ride> callback);
 
     /**
+     * Get receipt information for a completed request.
```

Comment:
Access to this endpoint is restricted and requires whitelisting. Could you add a sentence that says this?